docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,107 @@
+Scalar schema tool abstracts Request Unit of Cosmos DB with RU. So please set the appropriate value.
+
+#### Autoscale
+
+Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the throughput (RU/s) of your database or container automatically and instantly.
+The throughput is scaled based on the usage, without impacting the availability, latency, throughput, or performance of the workload.
+By default, the scalardl schema tool enables autoscale of RU for all tables: RU is scaled in or out between 10% and 100% of a specified RU depending on a workload.
+
+### Monitor Cosmos DB
+
+By default, monitoring is enabled on Cosmos DB.
+
+More information can be found in [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) documentation.
+
+## Azure Database for MySQL
+
+Azure Database for MySQL is a fully managed MySQL database provided by Azure. You can create a MySQL database by selecting a database instance and storage according to your configurations. The IOPS is calculated based on the compute instance and storage you choose.
+In this section, you will create an Azure Database for MySQL.
+
+### Recommendations
+
+* You should choose Flexible or Single server deployment for Azure Database for MySQL. You can check the [Azure documentation](https://docs.microsoft.com/en-us/azure/mysql/select-right-deployment-type) to learn more.
+* You should select a deployment that meets your IOPS requirements.
+    * For Single server deployments, IOPS is defined on the amount of storage provisioned(3 IOPS/GB).
+    * For Flexible server deployments, it is possible to scale IOPS and the maximum IOPS capacity will be defined by the amount of storage and the database instance chosen.
+
+### Steps
+
+For creating an Azure Database for MySQL, 
+* For Single Server deployment, follow this [Azure documentation](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal).
+* For creating a flexible server deployment, follow this [Azure documentation](https://docs.microsoft.com/en-us/azure/mysql/flexible-server/quickstart-create-server-portal).
+
+### Scale Performance
+
+**Scaling storage**:- The `storage auto grow` feature automatically increases storage when it reaches a threshold without impacting the workload.
+
+Resources can be scaled based on the deployment you select.
+* For Single server deployment, refer to the [Azure documentation](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#scale-resources) for scaling resources.
+* For Flexible server deployments, read more about scaling resources [here](https://docs.microsoft.com/en-gb/azure/mysql/flexible-server/concepts-compute-storage#scale-resources).
```

Comment:
Sorry, I feel it is not good. We can specify it like the above section.

Reply:
I will update it like the other sections.